packages/app/src/__deprecated__/renderer/styles/theming/theme.ts

```diff
@@ -44,6 +44,7 @@ const theme = {
       tabHover: grey1,
       actionHover: blue1,
       warning: orange,
+      info: grey1,
```

Comment:
We are sure that we would like to add a new color to text? 😬 I recommended getting a similar one from the existing one and giving feedback to a design team. 

Link to the Fonts section in our design system 👉 https://www.figma.com/file/82xELWhdkYtGeDQRkxYnf3/Center---Design-System?node-id=1%3A3&t=Nl9HvRhsCZQxZ17f-0

Reply:
This is the existing color from our design system (https://www.figma.com/file/82xELWhdkYtGeDQRkxYnf3/Center---Design-System?node-id=7%3A84&t=8xAbvJsOClb71HO8-4 - see `Grey 1`), the same as in `actionHover`. I didn't want to use `actionHover` for that - the name sounds inappropriate to what I wanted to achieve (just give the color to the text according to the design).

The above link does not specify text colors, but rather font types.

If you disagree, I can contact with design team and ask them to change the color, but seems to me that it follow our rules. What do you think?

Reply:
So maybe renaming `tabHover` will be a good compromise? 🤔